lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -125,6 +125,7 @@ class DOMSize extends Audit {
     return {
       score,
       numericValue: stats.totalBodyElements,
+      numericUnit: 'elements',
```

Comment:
`elements` isn't one of the [formattable unit types](https://tc39.es/proposal-unified-intl-numberformat/section6/locales-currencies-tz_proposed_out.html#sec-issanctionedsimpleunitidentifier), so I'm not clear on what this will be used for. Just for lhci-ui-type stuff (so [an assertion can print the type](https://github.com/GoogleChrome/lighthouse-ci/issues/100#issue-523898282))? If so, should we actually be using localized UIStrings for this field?

Reply:
> Just for lhci-ui-type stuff (so an assertion can print the type)?

Yeah any consumer that cares about *what in the world is this numeric value telling me*

> If so, should we actually be using localized UIStrings for this field?

I don't think so. This is basically an enum of a narrow set of things and it's up to the consumer to decide what to do with that information.

Reply:
IMO domsize is pretty darn unlike the rest of our audit return values.

I don't think it should report totalbodyElem as its numericValue. 
And also think that it should express its 3 numbers as numbers and not strings in the `.details`.

Thoughts?

Reply:
> I don't think it should report totalbodyElem as its numericValue.

I think this would be a major bummer for both assertions and the very purpose of `numericValue`. It's nifty that this is exposed this way and IMO is the only logical `numericValue` to accompany an audit called "DOM size". I might consider it the most appropriate use of `numericValue` in all of Lighthouse actually 😆 

> And also think that it should express its 3 numbers as numbers and not strings in the .details.

Totally agree it's cheating that it's using a `string` for `numeric` type already :)